content/redirect.js

```diff
@@ -278,7 +278,11 @@ const resolve = (url) => {
   if (!redirects.size) {
     load();
   }
-  return redirects.get(url.toLowerCase()) || resolveFundamental(url).url || url;
+  const fundamentalOrUrl = resolveFundamental(url).url || url;
```

Comment:
Probably a separate issue, but should `resolveFundamental` actually return the same URL even if it doesn't find a match instead of having to do this `||`'ing?

Reply:
I get your point. May that should be part of a big URL cleanup. Right now it's sometimes confusing if an URL is encoded/lowercased so returning and relying on `===` can be misleading. But I agree with the general issue at hand!